src/test/PoolTestBase.sol

```diff
@@ -17,25 +18,25 @@ abstract contract PoolTestBase is ILockCallback {
     }
 
     function _take(Currency currency, address recipient, int128 amount, bool withdrawTokens) internal {
-        assert(amount < 0);
+        assertGt(amount, 0);
```

Comment:
how come take just doesn't take in `uint128`

Reply:
yeah right now it just takes in an `int128` so that the tests can pass them in as like `delta.amount0()` etc without casting

Reply:
good call. same for settle.

Reply:
ya agreed. is there an issue open for this? Think this is a strictly good improvement. Want to make sure it doesnt get lost.

Reply:
its in the test improvements issue #390